### PR TITLE
add declarations for getX and getY in force directed graph

### PR DESCRIPTION
**Pull request by ehartford**
https://github.com/novus/nvd3/issues/1819

### CODE REVIEW

File: src/models/forceDirectedGraph.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ nv.models.forceDirectedGraph = function() {
         // These functions allow to add extra attributes to ndes and links
         ,nodeExtras = function(nodes) { /* Do nothing */ }
         ,linkExtras = function(links) { /* Do nothing */ }
+        , getX=d3.functor(0.0)
+        , getY=d3.functor(0.0)
```
Comment (liquidpele) [inline src/models/forceDirectedGraph.js]: getX and getY don't appear to actually be used anywhere?

Comment (ehartford) [inline src/models/forceDirectedGraph.js]: https://github.com/ehartford/nvd3/blob/dc81385b97ee48a7f8a9d8988f997867dc93b646/src/models/forceDirectedGraph.js#L165

Comment (liquidpele) [inline src/models/forceDirectedGraph.js]: ping

**Comment by liquidpele** [comment]
going to merge this just so calling x() and y() won't throw errors, but it looks like really getX and getY need to be either used or the whole options need to be removed entirely.

**Pull request merged**